src/styles/mixins/_tables.scss

```diff
@@ -236,20 +238,135 @@
             &.row-link {
               padding: var(--kui-space-0, $kui-space-0);
 
+              .cell-wrapper {
+                padding: var(--kui-space-50, $kui-space-50) var(--kui-space-60, $kui-space-60);
+              }
+
               a.cell-wrapper {
                 color: var(--kui-color-text, $kui-color-text);
                 display: block;
-                padding: var(--kui-space-50, $kui-space-50) var(--kui-space-60, $kui-space-60);
                 text-decoration: none;
               }
+
+              .expandable-row-control-container {
+                padding: var(--kui-space-50, $kui-space-50) var(--kui-space-60, $kui-space-60);
+              }
+            }
+
+            .expandable-row-control-container {
+              display: flex;
+
+              .expandable-row-control {
+                @include defaultButtonReset;
+
+                color: var(--kui-color-text-neutral, $kui-color-text-neutral);
+
+                &:hover,
+                &:focus,
+                &:focus-visible {
+                  color: var(--kui-color-text-neutral-strong, $kui-color-text-neutral-strong);
+                }
+
+                &:focus-visible {
+                  box-shadow: var(--kui-shadow-focus, $kui-shadow-focus);
+                }
+
+                .expandable-row-control-icon {
+                  height: $tableCaretIconSize !important;
+                  transition: transform $kongponentsTransitionDurTimingFunc;
+                  width: $tableCaretIconSize !important;
+                }
+
+                &.expanded {
+                  .expandable-row-control-icon {
+                    transform: rotate(90deg);
+                  }
+                }
+              }
+            }
+          }
+
+          &.last-row {
+            border-bottom: none;
+          }
+
+          &.expandable-content-row {
+            background-color: var(--kui-color-background-neutral-weakest, $kui-color-background-neutral-weakest);
```

Comment:
question: do we always want to enforce a background color?

Reply:
As per Figma designs, expanded row content has this background.